HabboHotel/Rooms/GameMap.cs

```diff
@@ -236,7 +236,7 @@ public void GenerateMaps(bool checkLines = true)
         }
         Array.Clear(tmpItems, 0, tmpItems.Length);
         tmpItems = null;
-        if (_room.RoomBlockingEnabled == false)
+        if (!_room.RoomBlockingEnabled == false)

```

Comment:
I'm not sure that's what @80O meant. I think he simply wanted you to use `if (!_room.RoomBlockingEnabled)` instead of `if (_room.RoomBlockingEnabled == false)`

Reply:
If I'm correct, same applies for each condition you changed

Reply:
This ^